app/assets/stylesheets/main.scss

```diff
@@ -12,6 +12,13 @@ table {
     font-weight: bold;
     margin-bottom: 0.75rem;
   }
+  td:has(.approved) {
+    text-align: center;
+    input[type="checkbox"] {
+      pointer-events: none;
+      accent-color: #059033;
+    }
+  }
```

Comment:
Not quite what I was talking about let's work on this together

Reply:
shoutout to using ":has" though. Pretty cool.